content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -147,7 +147,7 @@ different Kubernetes components.
 | `ServiceNodeExclusion` | `false` | Alpha | 1.8 | 1.18 |
 | `ServiceNodeExclusion` | `true` | Beta | 1.19 | |
 | `ServiceTopology` | `false` | Alpha | 1.17 | |
-| `SetHostnameAsFQDN` | `false` | Alpha | 1.19 | |
+| `SetHostnameAsFQDN` | `true` | Beta | 1.20 | |
```

Comment:
We retain the feature history for the alpha stage, so this should be;
```suggestion
| `SetHostnameAsFQDN` | `false` | Alpha | 1.19 | 1.19 |
| `SetHostnameAsFQDN` | `true` | Beta | 1.20 | |
```

Reply:
Sorry about that, I should have known.